dev/com.ibm.ws.ui.tool.serverConfig/resources/WEB-CONTENT/js/components/core.js

```diff
@@ -9,909 +9,753 @@
-            deferred.resolve(isServerStarted);
-        })
-        .fail(function() {
-            // If the controller cannot tell us if the server is running,
-            // then we should just assume the server status is unknown.
-            deferred.resolve(false);
-        });
-        return deferred;
-    };
-
-
-    var collectiveRoutingRequired = function() {
-        return environment === constants.ENVIRONMENT_COLLECTIVE && serverId !== hostServerId;
-    };
-
-
-    $(document).ready(function() {
-        
-        $(window).on("popstate", function() {
-            var skip = false;
-            if(embedded !== null && embedded !== undefined && window.top.location.hash.length === 0) {
-                skip = true;
-            }
-            if(!skip) {
-                if(embedded === constants.EMBEDDED_EXPLORE_TOOL) {
-                    var filePath = getDataFromURLHash().filePath;
-                    if(environment === constants.ENVIRONMENT_COLLECTIVE) {
-//                        window.top.history.pushState(null, null, "#" + constants.EXPLORE_TOOL_HASH + "/" + constants.EXPLORE_TOOL_SERVERS_SEGMENT + "/" + serverId + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + (filePath !== null && filePath !== undefined? "/" + filePath : ""));
-                        window.top.location.hash = "#" + constants.EXPLORE_TOOL_HASH + "/" + constants.EXPLORE_TOOL_SERVERS_SEGMENT + "/" + serverId + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + (filePath !== null && filePath !== undefined? "/" + filePath : "");
-                    } else {
-//                        window.top.history.pushState(null, null, "#" + constants.EXPLORE_TOOL_HASH  + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + (filePath !== null && filePath !== undefined? "/" + filePath : ""));
-                        window.top.location.hash = "#" + constants.EXPLORE_TOOL_HASH  + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + (filePath !== null && filePath !== undefined? "/" + filePath : "");
-                    }
-                } 
-                startup();
-            }
-        });    
-        
-        $(window).on("keydown", function(event) {
-            if(uiBlocked && event.keyCode !== 116) {
-                return false;
-            }
-        });
-        
-    });
-    
-
-    return {
-        messages: null,
-        startup: startup,
-        renderMessage: renderMessage,
-        showControlById: showControlById,
-        hideControlById: hideControlById,
-        setNavbarTitleText: setNavbarTitleText,
-        centerDialog: centerDialog,
-        setServer: setServer,
-        clearServer: clearServer,        
-        setFile: setFile,
-        clearFile: clearFile,
-        setUIBlocked: setUIBlocked,
-        getFileURL: getFileURL,
-        collectiveRoutingRequired: collectiveRoutingRequired,
-        getCollectiveServerId: function() {
-            return serverId;
-        },
-        isCollectiveServerRunning: function() {
```

Comment:
Why did we remove this? It seems like it's causing issues in a collective environment.

Reply:
After the validation changes, file was formatted in VS Code.
It looks like git is not supporting space changes while showing the code difference.
I will revert the file formatting changes.

Reply:
isCollectiveServerRunning() was removed during removal of old validator api, as that function was not getting called from any other files in serverConfig directory. This function is added back.